### PR TITLE
Tec 715

### PR DESCRIPTION
**Pull request by umbobabo**
Testable here: http://umbobabo.github.io/component-navigation/
I will remove a couple of fixes in a component-balloon refactor.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -277,7 +279,8 @@ export default class Navigation extends React.Component {
           />
           <div className="navigation__primary-expander" />
           <Button href={this.props.sharedMenu.subscribe.href}
-            className="navigation__main-navigation-link navigation__link navigation__main-navigation-link-subscribe"
+            className="navigation__main-navigation-link navigation__link navigation__main-navigation-link-subscribe
+            navigation__main-navigation--desktop"
```
Comment (keithamus) [inline src/index.js]: Why do we have so many classes for this? It seems excessive.

**Comment by fabiosantoscode** [comment]
Bravo, Umberto!

We have tappable area :)

**Comment by umbobabo** [comment]
@keithamus Can we merge?

**Comment by fabiosantoscode** [comment]
http://cdn.newsbusters.org/images/2013/June/Yes%20We%20Can.jpg

**Pull request merged**